dbos/utils_test.go

```diff
@@ -54,28 +54,26 @@ func setupDBOS(t *testing.T) {
 		t.Fatalf("failed to drop test database: %v", err)
 	}
 
-	err = Initialize(Config{
+	dbosContext, err := NewDBOSContext(Config{
 		DatabaseURL: databaseURL,
 		AppName:     "test-app",
 	})
 	if err != nil {
 		t.Fatalf("failed to create DBOS instance: %v", err)
 	}
-
-	err = Launch()
-	if err != nil {
-		t.Fatalf("failed to launch DBOS instance: %v", err)
-	}
```

Comment:
Most tests don't need Launch(). Only useful for queues, recovery thread, and notifications